### PR TITLE
[lighthouse] add reference for "rel=canonical" audit

### PR DESCRIPTION
**Pull request by kaycebasques**
What's changed, or what was fixed?
- adds reference doc for the `rel=canonical` SEO audit

**Fixes:** N/A

**Target Live Date:** 2018-03-13

- [x] This has been reviewed and approved by @rviscomi 
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/tools/lighthouse/_toc.yaml [hunk 1]
```diff
@@ -154,6 +154,8 @@ toc:
     path: /web/tools/lighthouse/audits/description
   - title: Document Doesn't Have A Title Element
     path: /web/tools/lighthouse/audits/title
+  - title: Document doesn't have a valid rel=canonical
```
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/_toc.yaml]: I'll eventually update all the titles to match the Lighthouse report formatting.

File: src/content/en/tools/lighthouse/audits/canonical.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+[LDU]: https://support.google.com/webmasters/answer/6080548
+
+## Recommendations {: #recommendations }
+
+Add a canonical link element to the `head` element of your HTML:
+
+    <!doctype html>
+    <html>
+      <head>
+        <link rel="canonical" href="https://example.com"/>
```
Comment (rviscomi) [inline src/content/en/tools/lighthouse/audits/canonical.md]: nit: rel=canonical doesn't necessarily need to be the first tag in the document head, and it's relatively important to optimize this ordering, so could you add another `...` before this?

Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/canonical.md]: ✔️

**Comment by kaycebasques** [comment]
Thanks @rviscomi, I think all your points are adequately handled now

**Pull request merged**